README.md

```diff
@@ -1,2 +1,15 @@
-# example-javascript
-Example app for using the DevCycle Client SDK with Javascript
+# DevCycle JavaScript SDK Example App
+
+An example app built using the [DevCycle JavaScript SDK](https://docs.devcycle.com/sdk/client-side-sdks/javascript/)
+
+## Running the example
+
+* Run `yarn install` in the project directory to install dependencies
+* Create a `.env` file and set `DEVCYCLE_CLIENT_SDK_KEY` to the SDK Key for your environment.\
+You can find this under [Settings > Environments](https://app.devcycle.com/r/environments) on the DevCycle dashboard.
+* [Create a new feature](https://docs.devcycle.com/introduction/quickstart#2-create-a-feature) on the dashboard with a string variable named `greeting` and a boolean variable named `spin-logo`
+* Open `index.html` to view it in your browser.
```

Comment:
they'll probably need to run `yarn build` before this step?

Reply:
oops good catch